packages/eslint-config-mc-app/package.json

```diff
@@ -22,7 +22,7 @@
   "module": "./index.js",
   "peerDependencies": {
     "babel-eslint": "10.x",
-    "eslint": "5.x",
+    "eslint": "5.x || 6.x",
     "eslint-config-airbnb-base": "13.x",
```

Comment:
Should we update this as well? If so, would that be a breaking change? 🤔

Reply:
I thought first we had to too but then I noticed that the `eslint-config-airbnb-base` has also both. I will adjust them to the exact same range?

https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/package.json#L68

Reply:
Ah yes thanks